docs/application/native/guides/device/settings.md

```diff
@@ -177,8 +177,10 @@ The following table lists the available system setting keys.
 | `SYSTEM_SETTINGS_KEY_3G_DATA_NETWORK_ENABLED` | `bool`   | `GETTER, NOTIFIER`      | Indicates whether the 3G data network is enabled. |
 | `SYSTEM_SETTINGS_KEY_ACCESSIBILITY_TTS`  | `bool`   | `GETTER, NOTIFIER`      | Indicates whether the accessibility TTS is enabled on the device. |
 | `SYSTEM_SETTINGS_KEY_ADS_ID`             | `string` | `GETTER, NOTIFIER`      | Ads ID for each device.                  |
+| `SYSTEM_SETTINGS_KEY_AUTOMATIC_TIME_UPDATE`  | `bool` | `GETTER, NOTIFIER`      | Updating time automatically.           |
```

Comment:
Updates time automatically.     


For clarity do you want consider saying; device time or system time something to make it more clear what time is updated?